src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCaller.java

```diff
@@ -197,7 +197,7 @@ public AssemblyRegionEvaluator assemblyRegionEvaluator() {
     @Override
     public void onTraversalStart() {
         final ReferenceSequenceFile referenceReader = getReferenceReader(referenceArguments);
-        hcEngine = new HaplotypeCallerEngine(hcArgs, getHeaderForReads(), referenceReader);
+        hcEngine = new HaplotypeCallerEngine(hcArgs, createOutputBamIndex, createOutputBamMD5, getHeaderForReads(), referenceReader);
```

Comment:
I hate having to pass around this pair of booleans everywhere.  It seems like it might be cleaner to have a little wrapper type, but maybe with 2 things it probably introduces more boilerplate than it solves.